package.json

```diff
@@ -94,5 +91,8 @@
         "react-dom"
       ]
     }
+  },
+  "dependencies": {
+    "fuels": "next"
```

Comment:
can we replace `next` with a specific version? using `next` is fragile

also applies for other dependencies using `next` across the project